### PR TITLE
LPS-122228 As an administrator, I want to edit a country's regions

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

File: modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/EditRegionMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+				SessionErrors.add(actionRequest, throwable.getClass());
+
+				actionResponse.setRenderParameter(
+					"mvcRenderCommandName", "/address/edit_region");
+			}
+			else {
+				throw new Exception(throwable);
+			}
+		}
```
Comment (drewbrokke) [inline modules/apps/address/address-web/src/main/java/com/liferay/address/web/internal/portlet/action/EditRegionMVCActionCommand.java:116]: We need to handle the following exceptions as well: 
`RegionCodeException`
`DuplicateRegionException`

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Looks good @pei-jung, just a couple of spots to address. See my inline comments.

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**